### PR TITLE
Use ES6 features

### PR DESCRIPTION
**Pull request by Turbo87**
Included in this PR are:

- Arrow functions
- Shorthand object methods and properties
- Template literals

### CODE REVIEW

**Comment by rwjblue** [comment]
@Turbo87 - Needs a rebase after https://github.com/ember-cli/ember-cli/pull/6631

File: .eslintrc.js [hunk 1]
```diff
@@ -89,8 +89,26 @@ module.exports = {
...
+    'quotes': [2, 'single', {
+      'allowTemplateLiterals': true,
+    }],
     'semi': [2, 'always'],
     'wrap-regex': 2,
+
+    /*** ECMAScript 6 ***/
+
+    'arrow-body-style': 2,
+    'arrow-parens': [2, 'as-needed'],
```
Comment (rwjblue) [inline .eslintrc.js]: I'd prefer to use the default here.

Comment (Turbo87) [inline .eslintrc.js]: 🚲 🚲 🚲 🚲 🚲 

I don't... 😜

**Comment by Turbo87** [comment]
@rwjblue rebased

**Comment by rwjblue** [comment]
@homu r+

**Pull request merged**